src/scenekit.cs

```diff
@@ -1715,10 +1972,19 @@ interface SCNRenderer : SCNSceneRenderer, SCNTechniqueSupport {
 		[iOS (9,0)][Mac (10,11)]
 		[Export ("renderAtTime:viewport:commandBuffer:passDescriptor:")]
 		void Render (double timeInSeconds, CGRect viewport, IMTLCommandBuffer commandBuffer, MTLRenderPassDescriptor renderPassDescriptor);
+
+		[Watch (4,0), TV (11,0), Mac (10,13), iOS (11,0)]
+		[Export ("renderWithViewport:commandBuffer:passDescriptor:")]
+		void RenderWith (CGRect viewport, IMTLCommandBuffer commandBuffer, MTLRenderPassDescriptor renderPassDescriptor);
```

Comment:
There are already two "Render()" methods that have been deprecated with no arguments.   I did not want to confuse those two, and could not come up a good reason to not use this name.